go/kbcmf/encrypt.go

```diff
@@ -135,12 +189,15 @@ func (es *encryptStream) init(sender BoxSecretKey, receivers [][]BoxPublicKey) e
 			gid = -1
```

Comment:
If the number of "receivers" groups is 2 because one of them represents the sender's devices, does that mean the sender is going to MAC for themselves?